builtin/game/statbars.lua

```diff
@@ -145,6 +145,7 @@ function core.hud_replace_builtin(hud_name, definition)
 	end
 
 	if hud_name == "health" then
+		definition.item = definition.item or core.PLAYER_MAX_HP_DEFAULT
```

Comment:
the definition table should probably be copied if you modify it

also shouldn't `item = number`?